src/components/drawer.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SwipeableDrawer from '@material-ui/core/SwipeableDrawer';
+import Button from '@material-ui/core/Button';
+
+const styles = {
+  list: {
+    width: 250,
+  }
+};
+
+
+class drawer extends React.Component {
+  constructor(props){
+      super(props);
+  };
+  
+
+  state = {
+    left: false,
+  };
+
+  toggleDrawer = (side, open) => () => {
+    this.setState({
+      [side]: open,
+    });
+  };
+
+  handleChange = name => event => {
+    this.setState({ [name]: event.target.checked });
+  };
+
+
```

Comment:
Minor: Can you change the consistency of line spacing to 1 line? Here and other places.